uvahelpr_web/marketplace/urls.py

```diff
@@ -6,4 +6,5 @@
     url(r'^$', views.index, name='index'),
     url(r'^jobs/(?P<id>[0-9]+)/$', views.job_entry, name="job_entry"),
     url(r'^jobs/$', views.allJobs, name='allJobs'),
+    url(r'^home/$', views.home, name='home'),
```

Comment:
There's also an index page, so it feels conflicting that there's two 'home' pages. Maybe combine both into one?